FlightReservation/components/termsComponents.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  TextInput,
+  Pressable,
+  Image,
+  TouchableOpacity,
+} from 'react-native';
+import CheckBox from '@react-native-community/checkbox';
+
+
+function Terms(props) {
+    const { changeForm, objValues } = props;
+    return (
+      <View style={[styles.containerForm, { marginTop: 30 }]}>
+        <View style={styles.checkboxContainer}>
+          <CheckBox
+            value={objValues.agreed}
+            onValueChange={(value) => changeForm('agreed', value)}
+            style={styles.checkbox}
+            tintColors={{ true: '#3E59F7', false: 'black' }}
+          />
+          <Text style={styles.labelCheckBox}>
+            I agree to the <Text style={{ textDecorationLine: 'underline' }}>Terms</Text> and <Text style={{ textDecorationLine: 'underline' }}>Privacy Policy.</Text>*
+          </Text>
+        </View>
+        <View style={styles.checkboxContainer}>
+          <CheckBox
+            value={objValues.subscribed}
+            onValueChange={(value) => changeForm('subscribed', value)}
+            style={styles.checkbox}
+            tintColors={{ true: '#3E59F7', false: 'black' }}
```

Comment:
line 24 and line 35 contains the same styling, please do a Stylesheet object and use it on those lines.